internal/modules/module.go

```diff
@@ -87,7 +90,20 @@ func New(_ context.Context, uri string, tr *configuration.TemplateRepository) (*
 		return nil, fmt.Errorf("version must be specified for module %q", tr.Name)
 	}
 
-	return &Module{template.New(tr.Name).Funcs(sprig.TxtFuncMap()), tr.Name, uri, tr.Version, nil}, nil
+	m := Module{
+		t:       template.New(tr.Name).Funcs(sprig.TxtFuncMap()),
+		Name:    tr.Name,
+		URI:     uri,
+		Version: tr.Version,
+	}
+
+	mani, err := m.getManifest(ctx)
```

Comment:
Honestly a smart move to fix `Manifest` to be set once. I don't remember my logic for making it the way I did, and it def caused friction.